docs/ref/templates/builtins.txt

```diff
@@ -694,6 +694,8 @@ includes it. This example produces the output ``"Hello, John!"``:
 * Template::
 
     {% include "name_snippet.html" %}
+    
+    Note that the above is referencing to an absolute path. If you need to look for ``name_snippet.html`` from the directory of the template referencing it, use ``{% include "name_snippet.html" %}``
```

Comment:
I'm not sure if "absolute path" is good terminology to use, considering that the given path is usually relative to the directory specified by a template loader. 

Formatting issues:
1. Lines must be wrapped at 80 characters.
2. This is on the same indent level as the code block and so will appear in it.
3. I guess you meant to use `"./name_snippet.html"` in the example.

A clarification might be better in the discussion of relative paths in the `{% extends %}` tag:
https://github.com/django/django/blob/d611a89238b0fe69ccb8a0e7bf20672015357128/docs/ref/templates/builtins.txt#L218-L232
For example, that could be revised to say something like "A string argument may be a relative path to the current template..."